NOTICE

```diff
@@ -1,5 +1,6 @@
 Druid - a distributed column store.
 Copyright 2012-2015 Metamarkets Group Inc.
+Copyright 2015 Yahoo! Inc.
```

Comment:
@xvrl thanks for the suggestion, but, wrt to licensing I'm just copy pasting stuff from internal email threads and update to NOTICE is found to be one acceptable way of doing it. but, anyway, let me ask @cheddar if he can get it resolved for us.

@cheddar can u pls take a look into this?